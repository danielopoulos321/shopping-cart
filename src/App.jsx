import { Outlet } from "react-router-dom";
import { Navbar } from "./components/Navbar/Navbar";
import styles from "./App.module.css";

function App() {
  const cartItems = [0, 0, 0, 0];
  return (
    <div className={styles.container}>
      <Navbar cartItems={cartItems} />
      <Outlet />
    </div>
  );
}

export default App;
