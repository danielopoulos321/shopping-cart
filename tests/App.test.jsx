/* eslint-disable no-undef */
import { render, screen } from "@testing-library/react";
import { describe, expect } from "vitest";
import App from "../src/App";

describe("App component", () => {
  it("render navbar and home on first load", () => {
    const { container } = render(<App />);
    expect(container).toMatchSnapshot();
  });
});
