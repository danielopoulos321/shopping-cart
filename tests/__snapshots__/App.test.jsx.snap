// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`App component > render navbar and home on first load 1`] = `
<div>
  <p>
    Loading...
  </p>
</div>
`;
